corehq/apps/domain/views.py

```diff
@@ -729,21 +728,19 @@ def get_feature_summary(self, plan_version, account, subscription):
                     if feature_rate.monthly_limit != -1
                     else _('Unlimited')
                 ),
-                'credit': self._fmt_credit(),
                 'type': feature_rate.feature.feature_type,
                 'recurring_interval': get_feature_recurring_interval(feature_rate.feature.feature_type),
             }
 
-            credit_lines = None
-            if subscription is not None:
-                credit_lines = CreditLine.get_credits_by_subscription_and_features(
-                    subscription, feature_type=feature_rate.feature.feature_type
-                )
-            elif account is not None:
-                credit_lines = CreditLine.get_credits_for_account(
-                    account, feature_type=feature_rate.feature.feature_type)
-            if credit_lines:
-                feature_info['credit'] = self._fmt_credit(self._credit_grand_total(credit_lines))
+            total_feature_credit = CreditLine.get_total_credits(
+                subscription=subscription,
+                account=account,
+                feature_type=feature_rate.feature.feature_type,
+            )
+            if total_feature_credit is not None:
```

Comment:
I think this `if` statement is redundant, since I think calling `self._fmt_credit()` is equivalent to calling `self._fmt_credit(total_feature_credit)` if `total_feature_credit` is `None`.